cdp_client/cdp.py

```diff
@@ -1,6 +1,7 @@
 from promise import Promise
 from time import sleep
 from collections import namedtuple
+from hashlib import sha256
```

Comment:
is this part of standard python? if not then dependencies go to setup.py to be downloaded automatically

Reply:
IMO it is standard pyhton?

Reply:
depends if you installed it with pip install or not, maybe it is

Reply:
I haven't installed it intentionally. Neither list my pip list installed it.